README.md

```diff
@@ -24,44 +22,104 @@ The repository structure follows the conceptual architecture of Snowplow, which
-$ cd snowplow/3-enrich/scala-common-enrich
-$ sbt test
-```
+Components that have been extracted to their own repo are still here as [git submodules][submodules].
 
-## Find out more
+Please use directly the repo of a component to report issues and create PRs:
 
-| **[Technical Docs][techdocs]**     | **[Setup Guide][setup]**     | **[Roadmap][roadmap]**           | **[Contributing][contributing]**           |
-|-------------------------------------|-------------------------------|-----------------------------------|---------------------------------------------|
-| [![i1][techdocs-image]][techdocs] | [![i2][setup-image]][setup] | [![i3][roadmap-image]][roadmap] | [![i4][contributing-image]][contributing] |
+### Trackers
 
-## Contributing
+- [Android](https://github.com/snowplow/snowplow-android-tracker)
+- [.NET](https://github.com/snowplow/snowplow-dotnet-tracker)
+- [Go](https://github.com/snowplow/snowplow-golang-tracker)
+- [JAVA](https://github.com/snowplow/snowplow-java-tracker)
+- [Javascript](https://github.com/snowplow/snowplow-javascript-tracker)
+- [Node.js](https://github.com/snowplow/snowplow-nodejs-tracker)
+- [Objective-C](https://github.com/snowplow/snowplow-objc-tracker)
+- [PHP](https://github.com/snowplow/snowplow-php-tracker)
+- [Python](https://github.com/snowplow/snowplow-python-tracker)
+- [Ruby](https://github.com/snowplow/snowplow-ruby-tracker)
+- [Scala](https://github.com/snowplow/snowplow-scala-tracker)
+- [Unity](https://github.com/snowplow/snowplow-unity-tracker)
 
-We're committed to a loosely-coupled architecture for Snowplow and would love to get your contributions within each of the six sub-systems.
+### [Collector](https://github.com/snowplow/stream-collector)
 
-If you would like help implementing a new tracker, adding an additional enrichment or loading Snowplow events into an alternative database, check out our **[Contributing][contributing]** page on the wiki!
+### [Enrich](https://github.com/snowplow/enrich)
 
-## Questions or need help?
+### Loaders
 
-Check out the **[Talk to us][talk-to-us]** page on our wiki.
+- [BigQuery (streaming)](https://github.com/snowplow-incubator/snowplow-bigquery-loader)
+- [Redshift / Postgres (batch)](https://github.com/snowplow/snowplow-rdb-loader)
+- [Snowflake (batch)](https://github.com/snowplow-incubator/snowplow-snowflake-loader)
+- [Google Cloud Storage (streaming)](https://github.com/snowplow-incubator/snowplow-google-cloud-storage-loader)
+- [Amazon S3 (streaming)](https://github.com/snowplow/snowplow-s3-loader)
+- Postgres (streaming)(https://github.com/snowplow-incubator/snowplow-postgres-loader)
+- [Elasticsearch (streaming)](https://github.com/snowplow/snowplow-elasticsearch-loader)
+
+### Testing
+
+- [Mini](https://github.com/snowplow/snowplow-mini)
+- [Micro](https://github.com/snowplow-incubator/snowplow-micro)
+
+### Parsing enriched event
+
+- [Analytics SDK Scala](https://github.com/snowplow/snowplow-scala-analytics-sdk)
```

Comment:
There is also a dotnet SDK and a JavaScript SDK.

https://github.com/snowplow/snowplow-dotnet-analytics-sdk
https://github.com/snowplow-incubator/snowplow-js-analytics-sdk/